_posts/2022-03-08-using-the-rac-collections-api-to-create-visualizations.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Using the RAC's Collections API to create visualizations "
+date: 2022-03-08T09:00:00
+author: Hillel Arnold
+layout: post
+categories:
+  - Collaboration
+tags:
+  - api
+  - visualizations
+  - collaboration
+  - expertise
+excerpt_separator: <!--more-->
+---
+Over the past few months, I’ve been experimenting with our Collections API to explore its possibilities. Most recently, I’ve used it to create some visualizations which provide a visual avenue for exploration of collections, as well as that might not be possible simply from reading the description. Now that we have publicly launched the Collections API, I wanted to write about this project as an example of how the API provides some new ways of exploring the RAC’s collections. <!--more-->
```

Comment:
Maybe we should link to the API documentation either in the first or second paragraph? (docs.rockarch.org/argo)

Reply:
Linked to the API in the first paragraph and added a parenthetical reference to the docs which I also linked.